vic/drivers/shared_image/src/vic_image_timing.c

```diff
@@ -70,6 +72,10 @@ write_vic_timing_table(timer_struct *timers,
         strcpy(user, "unknown");
     }
 
+    // mpi/openmp
+    nthreads = omp_get_max_threads();
```

Comment:
What happens here when the code is *not* compiled with the `openmp` library? Seems to me that we are requiring openmp to be present and available even if we don't use it (just like we do for mpi). If that is the case, please make that clear in the release notes and in the documentation.